### PR TITLE
Add documentation for installation in Eclipse and IntelliJ

### PR DESCRIPTION
**Pull request by Entenwilli**
This PR adds instructions in the README.md to set up the data flow analysis with both eclipse and IntelliJ

### CODE REVIEW

**Comment by Entenwilli** [comment]
@sebinside @uuqjz @Nicolas-Boltz @01Parzival10 @BenjaminArp Some feedback on this would be nice. Should I add some pictures for each step?

**Comment by 01Parzival10** [comment]
I managed to run the analysis ba referencing it from a new Project.
However, I can seem to get existing projects to run. Eclipse does not recognize the main/test classes

**Comment by Entenwilli** [comment]
> I managed to run the analysis by referencing it from a new Project. However, I can seem to get existing projects to run. Eclipse does not recognize the main/test classes

Do you mean IntelliJ, or really Eclipse?

**Comment by Entenwilli** [comment]
I've updated the guide by detailing a few more steps and images. This should hopefully fix it.
Nonetheless, this is valuable feedback, as we should expect users to use the DFA from these instructions

**Comment by Entenwilli** [comment]
@01Parzival10

**Review by sebinside — CHANGES_REQUESTED** [review]
I tried to follow the IntelliJ guide but without success. Here are a couple of comments / hints:
* The guide lack an overview that tells less experienced developers (or developers that don't know IntelliJ at all), which steps are required in general
* Many steps lack details are us ambiguous language, e.g., step 2 with "our data flow analysis product" (which product, extracted zip?), "well known location" (which location, does it matter, or just that I know where I put it)
* The pictures help a lot, but are sometimes not enough, e.g., step 5 where you expect the "+" button to be clicked. Also other steps sometimes lack precise information where to look or what to click
* I was not able to understand neither follow step 8 - 11
* The guide ends without showing a final results or showing how to, e.g., execute a test

TL;DR: In its current version, the guide requires a lot of detail knowledge that it should explain to non-experts in the first place :)

**Comment by Entenwilli** [comment]
I've rephrased the installation instructions. Some tasks are still open:
- [ ] Mention IntelliJ verison 

For which step should I insert more images?

**Comment by BenjaminArp** [comment]
Setup IntelliJ with nightly and current IntelliJ version in 20 minutes without any issues.

**Comment by Entenwilli** [comment]
Superseeded by #292
